vector/src/main/java/im/vector/app/features/media/ImageContentRenderer.kt

```diff
@@ -117,7 +117,7 @@ class ImageContentRenderer @Inject constructor(private val activeSessionHolder:
                     .load(resolvedUrl)
         }
 
-        req.override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL)
+        req
```

Comment:
Will this change reduce the quality of the image and so even when the image is pinched (zoomed in)?

Reply:
Yes this is important, we ideally want the original image. This call is used for the image viewer that has the zoom feature. 
But eventually we only need maxZoom * screen size?
Not sure exactly how to use it but try to see sizeMultiplier() ? We don't have target size here, but we could override to screen size.
Maybe it's too much memory for screen of very high density?